tests/test_blockUtils.py

```diff
@@ -67,8 +67,7 @@ def getBlockInfoTestTruthValues(dataFilename=None):
         The block info truth data.
     """
     if dataFilename is None:
-        packageDir = getPackageDir("summit_utils")
-        dataFilename = os.path.join(packageDir, "tests", "data", "blockInfoData.json")
+        dataFilename = os.path.join(TESTDIR, "data", "blockInfoData.json")
```

Comment:
Is the package dir method no longer preferred? I don't see a problem with this switch, but just want to educate myself, as this method used to be the way to do it, and I'm curious why changed, and if I should change it everywhere.

Reply:
If you are looking for test data in a subdir relative to the test file, you should not use `getPackageDir` because you always want the test data files to go with the test python files regardless of how EUPS has been configured. This is future-proofed as well since it lets tests run in a future that doesn't always have EUPS.

You currently have to use `getPackageDir` in the package code to locate configs but at some point we really do need to move configs inside the python tree so we can use standard python package data to find the configs (which would let them work even without EUPS) but that is a big change that needs an RFC and changes to pex_config.

Reply:
OK, sounds good, thanks.